### DISCUSSION TITLE
Can't download ugs platform 2.0.8 for raspberry pi

### DISCUSSION THREAD

**Discussion by jrdarrah** [winder/Universal-G-Code-Sender#1677] [Help]
How do I download the 2.0.8 platform version for raspberry pi? I went here  [https://github.com/winder/Universal-G-Code-Sender/releases]     to download it and I get this error.

{
  "errors": [
    {
      "status": 404,
      "message": "Item UGS:v2.0.8/ugs-platform-app-pi.tar.gz does not exist"
    }
  ]
}

**Comment by jrdarrah** [comment]
I found the google groups so I'm moving the question there

**Reply by breiler** [reply]
It looks like the binary is missing for that version, I would recommend the latest nightly build.
